glean_parser/templates/markdown.jinja2

```diff
@@ -28,9 +28,30 @@ See the Glean SDK documentation for the [`{{ ping_name }}` ping]({{ ping_name|pi
 {% if ping_name|ping_send_if_empty %}
 This ping is sent if empty.
 
+{% endif %}
+{% if ping_name|ping_include_client_id %}
+This ping includes the [client id](https://mozilla.github.io/glean/book/user/pings/index.html?highlight=client_info#the-client_info-section).
+
+{% endif %}
+{% if ping_name|ping_data_reviews %}
+**Data reviews for this ping:**
+
+{% for review in ping_name|ping_data_reviews %}
+    - <{{review}}>
+{% endfor %}
+
+{% endif %}
+{% if ping_name|ping_bugs %}
+**Bugs related to this ping:**
+
+{% for bug in ping_name|ping_bugs %}
+    - {% if bug|int != 0 %}{{bug}}{% else %}<{{bug}}>{% endif %}
```

Comment:
Linking the bug number to BMO would be the best usability here... but on the other hand, let's not make bug numbers (a thing we deprecated) any nicer than they have to be : )

Reply:
My decision to not do that was because it is not stated that if this is a number, it will be a Bugzilla bug number.

Reply:
> 
> 
> My decision to not do that was because it is not stated that if this is a number, it will be a Bugzilla bug number.

Yes, and that's a good call. Fenix used to have issue numbers there, that's why we moved to requiring URIs.